cve/system/ksp-npm-gruntjs-cve-2020-7729.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: security.kubearmor.com/v1
```

Comment:
@rohitrishim please add the kubearmor banner also
cc: @salman-accuknox @raviknox